opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -64,8 +64,7 @@
 from contextlib import contextmanager
 import typing
 
-from opentelemetry import loader
-from opentelemetry import types
+from opentelemetry import loader, types
```

Comment:
Please don't import multiple modules in a single line.

Reply:
@toumorokoshi Since you approved the PR but made this comment, I'm not sure what's your opinion on that code? Do you prefer importing multiple submodules on the same line or each on their own line.

Ad linters: I think pylint knows quite a bit about the meaning of these names. There is no check against it though.

Reply:
My opinion isn't strong enough for me to argue the point, but I actually prefer one per line. My personal favorite way to refer to an import is Rust syntax, which enables a lot of flexibility with a visually intuitive ident-as-namespace:

https://doc.rust-lang.org/reference/items/use-declarations.html

I think for those who don't use an ide, importing modules exclusively is really helpful. It makes it easy to see what belongs to a module vs what is a local reference:

```
foo = Foo()
other_object = other_module.Bar()
```

Once I know it's referenced locally or imported, I then make the determination to go to the top of the file, or to search for the next reference. But I use the go-to-definition functionality of my ide, and so the organization doesn't affect me either away.

The only thing I really dislike is star imports. But I think most people agree on that.

Reply:
@Oberon00 it looks like the only way to use `isort` consistently with `black` is to allow grouped imports for module names.